README.md

```diff
@@ -30,7 +30,7 @@ type: Opaque
 data:
   key: APIKEY_BASE64
 ---
-apiVersion: certmanager.k8s.io/v1alpha1
+apiVersion: certmanager.k8s.io/v1alpha3
```

Comment:
With the above, `error: unable to recognize "config.yaml": no matches for kind "Issuer" in version "certmanager.k8s.io/v1alpha3"`

I think this needs to be `cert-manager.io/v1alpha3`?

Reply:
yes, It's fixed :)